### DISCUSSION TITLE
Switch example to control relays

### DISCUSSION THREAD

**Discussion by MarianTudor82** [dawidchyrzynski/arduino-home-assistant#94] [Q&A]
Hello! Thank you for your efort to create this useful library! Since I didn’t have to much experience with HA and Arduino, I kindly ask you for help with the following scenario/config:
Hardware: Arduino uno  + Ethernet Shield & 4 Relay Board with 4 channels
Sensors: DHT11
Scenario: use the Genetic Thermostat from HA to control the actuators of Underfloor heating. 

So for me it will be very useful if you can provide an example like: when the button from HA UI will be press, the switch (relay) will TURN ON…
Thank you!

**Comment by dawidchyrzynski** [comment]
Hi @MarianTudor82,

There is already an example with implementation for a relay: https://github.com/dawidchyrzynski/arduino-home-assistant/blob/main/examples/led-switch/led-switch.ino
